api_tests/providers/registrations/views/test_registration_provider_schemas.py

```diff
@@ -100,9 +102,10 @@ def test_registration_provider_with_schema(
         assert res.status_code == 200
         data = res.json['data']
 
-        assert len(data) == 1
-        assert data[0]['id'] == schema._id
-        assert data[0]['attributes']['name'] == schema.name
+        assert len(data) == 2
+        assert schema._id in [item['id'] for item in data]
+        assert invisible_schema._id in [item['id'] for item in data]
+        assert schema.name in [item['attributes']['name'] for item in data]
 
         res = app.get(url_with_v2_prereg_only, auth=user.auth)
```

Comment:
The user for the requests in this test needs to be an `egap_admin`. That should bring about the right conditions for the fix that needs to go in.